packages/mobile/src/fiatExchanges/ReviewFees.tsx

```diff
@@ -66,7 +66,11 @@ export default function ReviewFees({
     />
   )
 
-  const token = currencyToBuy === CiCoCurrency.CUSD ? 'cUSD' : 'CELO'
+  const token = {
+    [CiCoCurrency.CELO]: 'CELO',
+    [CiCoCurrency.CUSD]: 'cUSD',
+    [CiCoCurrency.CEUR]: 'cEUR',
```

Comment:
can you add cREAL so we don't have this silly mobile update to do when/if that happens?